### PR TITLE
Microfulproject

### PR DESCRIPTION
**Pull request by obibaadoma**
Made Changes to HTML
Changes Made to CSS to

### CODE REVIEW

**Comment by obibaadoma** [comment]
Changes Made to HTML and CSS

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
   <section class="headline">
     <div id="head-container">
       <h1 class="head">
-        I'm Derek, <br />
+        I'm Derek, <br/>
```
Comment (franclobo) [inline index.html:45]: - [ ] Kindly add the corresponding background image to the home page 👇🏽 

image

Comment (franclobo) [inline index.html:45]: - [ ] Please 🙏🏽 use the correct font styles in the title:

image

File: index.html [hunk 2]
```diff
@@ -88,7 +88,7 @@ <h2 class="connect">LET’S CONNECT</h2>
       <img class="snapshort" src="./images/snap1.png" alt="snapshoot" />
       <div class="left-block-tonic">
         <div class="first-container-tonic">
-          <h1 class="newme-title-tonic">Tonic</h1>
+          <h2 class="newme-title-tonic">Tonic</h2>
```
Comment (franclobo) [inline index.html:91]: - [ ] As I said before 👆🏽 please 🙏🏽 use the correct font styles to be similar to the Figma template. In general review the styles of the fonts and also the buttons.

image

File: index.html [hunk 3]
```diff
@@ -128,11 +128,11 @@ <h3>2015</h3>
       <img class="snapshort" src="./images/snap2.png" alt="snapshoot" />
       <div class="left-block">
         <div class="first-container">
-          <h1 class="newme-title">
-            Multi-Post <br />
-            Stories
-          </h1>
-          <div class="event2">
+          <h2 class="newme-title">
```
Comment (franclobo) [inline index.html:131]: - [ ] As I said before 👆🏽 please 🙏🏽 use the correct font styles to be similar to the Figma template. In general review the styles of the fonts and also the buttons.

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 4]
```diff
@@ -171,7 +171,7 @@ <h3>2015</h3>
       <img class="snapshort" src="./images/snap3.png" alt="snapshoot" />
       <div class="left-block-tonic">
         <div class="first-container-tonic">
-          <h1 class="newme-title-tonic">Tonic</h1>
+          <h2 class="newme-title-tonic">Tonic</h2>
```
Comment (franclobo) [inline index.html:174]: - [ ] As I said before 👆🏽 please 🙏🏽 use the correct font styles to be similar to the Figma template. In general review the styles of the fonts and also the buttons.

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 5]
```diff
@@ -207,14 +207,15 @@ <h3>2015</h3>
...
+    <section class="works-card-tonic">
       <img class="snapshort" src="./images/snap4.png" alt="snapshoot" />
       <div class="left-block">
         <div class="first-container">
-          <h1 class="newme-title">
-            Multi-Post <br />
-            Stories
-          </h1>
+          <h2 class="newme-title">
+            Multi-Post Stories<br />
```
Comment (franclobo) [inline index.html:216]: - [ ] As I said before 👆🏽 please 🙏🏽 use the correct font styles to be similar to the Figma template. In general review the styles of the fonts and also the buttons.

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 6]
```diff
@@ -254,10 +255,10 @@ <h3>2015</h3>
   <section class="contact-me">
     <div class="head-section-co">
       <div id="head-container-co">
-        <h1 class="head-co">
+        <h2 class="head-co">
           About, <br />
           Myself
-        </h1>
+        </h2>
```
Comment (franclobo) [inline index.html:261]: - [ ] As I said before 👆🏽 please 🙏🏽 use the correct font styles to be similar to the Figma template. In general review the styles of the fonts and also the buttons.

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @obibaadoma, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] The contact form is not required at this moment, please 🙏🏽 remove it.

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**